doc/architecture/decisions/0004-reuse-api-for-cas1-and-cas3.md

```diff
@@ -0,0 +1,85 @@
+# 4. reuse-api-for-cas1-and-cas3
```

Comment:
sorry but "Generate Domain Events..." (https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/156) has snuck in ahead of this in position 4